sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -331,7 +349,7 @@ export class MessageSender extends LinkEntity {
       operation: sendEventPromise,
       connectionId: this._context.namespace.connectionId!,
       operationType: RetryOperationType.sendMessage,
-      retryOptions: {
+      retryOptions: options.retryOptions || {
```

Comment:
Does this need to be more granular? Ie, I can toggle just one or the other setting and take the default for whatever I didn't set?

Reply:
Hmm.. makes sense.